README.md

```diff
@@ -94,98 +93,116 @@ Commands:
+  -h, --help         show help                                                                                                                   [boolean]
 
 Status Codes:
   Success (0)       Command successfully completed
   Error (1)         An error was encountered while executing command
   Cancelled (130)   User responded 'No' to iidy prompt or interrupt (CTRL-C) was received
 ```
 
-### The `argsfile` (aka `stack-args.yaml`)
+### Environment Variables
+
+Any parameter used by iidy can be set using `IIDY_{{argname}}`. An example of
+this would be chaning the default environment from development to production.
+
+```shell
+export IIDY_ENVIRONMENT=production
+```
 
-```YAML
-####################
-# Required settings:
+### The Args File
 
-StackName: <string>
+Metadata for a CloudFormation stack is kept in a file called the "argsfile"
+(typically called `stack-args.yaml`). This file is a parameter for commands like
+`iidy create-or-update` or `iidy create-changeset` and is the main source of
+data for creating or updating a CloudFormation stack.
 
-Template: <local file path or s3 path>
+#### Required Properties
 
-# optionally you can use the YAML pre-processor by prepending 'render:' to the filename
-#Template: render:<local file path or s3 path>
+| Property | Description | Example |
+|----------|-------------|---------|
+| StackName | CloudFormation stack name | `my-stack` |
+| Template | Local, `https` or `s3` location of CloudFormation template | `cfn-template.yaml` |
 
+#### Optional Properties
 
-####################
-# Optional settings:
+| Property | Description | Example |
+|----------|-------------|---------|
+| Region | AWS Region to use, overridden by `--region` and `AWS_REGION` | `us-east-1` |
+| Profile | [AWS profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) to use, overridden by `--profile` and `AWS_PROFILE` |
+| AssumeRoleARN | IAM Role ARN to assume | `arn:aws:iam::1234567890:role/my-role` |
+| ServiceRoleARN | [CloudFormation Service Role](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-iam-servicerole.html) to use |
+| Tags | Tags to assign to CloudFormation stack | `{ service: my-app }` |
+| Parameters | [CloudFormation stack parameters](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html) to use |
+| Capabililites | List of [CloudFormation capabilities to use](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-iam-template.html#using-iam-capabilities) | `CAPABILITY_NAMED_IAM` |
+| NotificationARNs | List of [SNS Topic ARNs to send CloudFormation notifications to] | `arn:aws:sns:us-east-1:123467890:my-topic` |
+| TimeoutInMinutes | Number of minutes to provide as [timeout to the CloudFormation service](https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateStack.html) | `10` |
```

Comment:
Does this have a default value?

Reply:
There is no default value specified in iidy. Also:

> By default, there is no timeout for stack creation. However, individual resources may have their own timeouts based on the nature of the service they implement.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-add-tags.html